app/views/case_workers/admin/management_information/_stats_report.html.haml

```diff
@@ -10,10 +10,18 @@
   %p
     = t('unavailable_report', scope: i18n_scope)
 
-- if report_rec&.started_at.present?
+- if report_rec&.started_at.present? && report_object.updatable?
```

Comment:
How does this work if both the `if` are inside the `govuk-button-group`? This will mean that single buttons are inside a group in the case when one or the other isn't required but I don't think that is a problem.

Reply:
@jrmhaig -I have a memory of attempting it like that and the formatting went a bit strange but now it seems to work! So yes that looks so much better. Thank you Joe!